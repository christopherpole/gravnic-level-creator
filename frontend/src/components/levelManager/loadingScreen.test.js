import React from 'react';
import { configure, shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import LoadingScreen from './loadingScreen';

configure({ adapter: new Adapter() });

describe('<LoadingScreen />', () => {
  it('Renders without exploding', () => {
    const loadingScreen = shallow(<LoadingScreen />);

    expect(loadingScreen).toHaveLength(1);
  });

  it('Matches the current snapshot', () => {
    const loadingScreen = shallow(<LoadingScreen />);

    expect(toJson(loadingScreen)).toMatchSnapshot();
  });
});
