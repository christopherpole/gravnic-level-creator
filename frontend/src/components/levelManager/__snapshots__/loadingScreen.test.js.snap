// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The loading screen Matches the current snapshot 1`] = `
<styled.div>
  <LoadingIcon
    fillColor="white"
    size={40}
    speed={1}
  />
</styled.div>
`;
